RxJavaTemplate/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath("com.android.tools.build:gradle:${Versions.BUILD_GRADLE_VERSION}")
+        classpath("com.google.dagger:hilt-android-gradle-plugin:${Versions.HILT_VERSION}")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.KOTLIN_VERSION}")
+        classpath("androidx.navigation:navigation-safe-args-gradle-plugin:${Versions.NAVIGATION_VERSION}")
```

Comment:
can't these paths be managed in the same manner as you did for the Dependencies declaration? 
`"androidx.navigation:navigation-safe-args-gradle-plugin:` -> `ANDROIDX_SAFE_ARGS_PLUGIN`? 🤔

Reply:
@sleepylee The configuration for classpath will be updated in the next upcoming PR. On this PR's scope, I am doing migration rather than refactoring the way we organize dependencies 🙏